analyzers/its/.gitignore

```diff
@@ -1,6 +1,3 @@
-# SonarLint binaries
```

Comment:
I'd be also fine with deleting this whole file and moving its content to the root. To manage these settings on a single place.